contracts/bond/BondMediator.sol

```diff
@@ -35,31 +37,35 @@ contract BondMediator is
      * @param factory A deployed BondCreator contract to use when creating bonds.
      * @param manager A deployed BondCurator contract to register created bonds with,
      * @param erc20CapableTreasury Treasury that receives forfeited collateral. Must not be address zero.
+     * @param erc20CollateralTokens Collateral token contract. Must not be address zero.
      */
     function initialize(
         address factory,
         address manager,
-        address erc20CapableTreasury
+        address erc20CapableTreasury,
+        address erc20CollateralTokens
```

Comment:
should this be singular? erc20CollateralToken?

Reply:
Good question, it depends on the context. 
When passing around the address or IERC20 (or equivalent) the reference is to the token contract, which being the manager of the token, is plural.

This naming convention also carries over to the tests, where the variable name for the IERC20 contracts are plural 😁 

There's a number of OS cases, the one that springs to mind (and quickly found 😉 )  
https://github.com/jbx-protocol/juice-contracts-v2/blob/fd289805dbca5cc8c5677ef92e5461fe57f928d7/contracts/JBETHPaymentTerminal.sol#L246